backend/routes_calibration.py

```
@@ -80,17 +89,39 @@ def get_calibrations_due():
             threshold_date = now + timedelta(days=days)
 
             # Find tools that require calibration and are due within the specified days
+            # Use the calibration_status field to ensure consistency across the application
             tools = Tool.query.filter(
                 Tool.requires_calibration == True,
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if Tool.requires_calibration:` for truth checks

Replace with `Tool.requires_calibration`

(E712)